### DISCUSSION TITLE
Tesorflow with cuda support

### DISCUSSION THREAD

**Discussion by nandra** [OE4T/meta-tegra#519] [Q&A] [answered]
Just curious if anybody is working on tensorfow support in Yocto with cuda support. Thanks.

**Comment by dwalkes** [comment, marked as answer]
My relatively inexperienced stint with tensorflow-cc + cuda has felt a lot like swimming upstream and I wouldn't recommend it to anyone unless they happen to enjoy self punishment. That said I do have some instructions at https://github.com/BoulderAI/docker-tensorflow-l4t and a [container on docker hub](https://hub.docker.com/r/baicontainers/l4t-tensorflow-cc) which could be useful, especially if you happen to be working with JP 4.4 and tensorflow 1.15 and can run your application in a container.

**Reply by dat-tran-matternet** [reply]
What framework do you recommend if not tensorflow for a beginner to start with and be able to leverage the hardware acceleration on a TX2 or Xavier NX? Thanks
